Makefile

```diff
@@ -127,11 +161,41 @@ preload-images:
 		import_image quay.io/submariner/$${image}; \
 	done
 
+# Generate kustomization.yaml for bundle
+kustomization: manifests
```

Comment:
Does this need to be included in the release process somehow? I ask because when I run `make kustomization`, I get a modified version of `config/manager/kustomization.yaml`.

```
-  newTag: 0.7.0
+  newTag: v0.7.0-16-g80b6c81-dev
```

Reply:
do you mean the new release repo? then yes.
`make bundle` and `make packagemanifests` both create a new bundle in different formats
and both use `make kustomization` as a preparation step

Reply:
Thanks @SteveMattar, that makes sense. :+1: 

Are there any updates we need to make to https://submariner.io/contributing/release-process/?

Reply:
@dfarrell07 yes in [step 9](https://submariner.io/contributing/release-process/#step-9-update-the-submariner-operator-on-operatorhubio) I have already prepared a commit locally for the submariner-website but waiting for this one to be merged first

Reply:
Imho it would be best to go ahead and push it, mark as draft, and we can start reviewing but hold on merging until this lands.

Reply:
ACK